src/com/palmergames/bukkit/towny/huds/PermHUD.java

```diff
@@ -33,6 +32,14 @@ public static void updatePerms (Player p) {
 	public static void updatePerms(Player p, WorldCoord worldCoord) {
 		String plotName, build, destroy, switching, item, pvp, explosions, firespread, mobspawn, title;
 		Scoreboard board = p.getScoreboard();
+		// Due to dick delay (probably not confirmed), a HUD can actually be removed from the player.
```

Comment:
```suggestion
		// Due to tick delay (probably, not confirmed), a HUD can actually be removed from the player.
```

Reply:
I've corrected this an amended it. good catch xD